distribution/test/itests/README.md

```diff
@@ -14,14 +14,20 @@ It is possible to SSH into a running test instance. This will allow you to use t
 ssh -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no -p 20003 admin@localhost
 ```
 
-## Debug a Single Test
+## Debugging Tests
```

Comment:
Now that I think about it, this section is more about "Running Tests" than debugging them. It sort of implies that these commands can only be used for debugging when that is not the case. I'd suggest renaming it